pshops_lib/main_menu_display.js

```diff
@@ -0,0 +1,12 @@
+/*
+    Function: main_menu_display.js
+    Purpose: displays main menu for the PShop client portal
+*/
+
+module.exports = function(farmer_name){
```

Comment:
move this into your main module - abstracting this out is hard to follow and may have unintended consequences